src/dlgConnectionProfiles.cpp

```diff
@@ -716,166 +716,311 @@ QPair<bool, QString> dlgConnectionProfiles::writeProfileData(const QString& prof
+                   "own heroes, villains, battles, and endings. Over the course of each two-year "
+                   "timeline, the game explores all the key eras of the Star Wars Expanded Universe."
+                   "\n\n"
+                   "Take and hold planets as an Imperial Stormtrooper, command the Rebel navy and "
+                   "liberate the galaxy, pursue targets as a bounty hunter, or shape things on a "
+                   "larger scale as a member of the Galactic Senate. Maybe you'll even be one of the "
+                   "few born with force sensitivity, destined to be trained by Jedi or Sith."
+                   "\n\n"
+                   "The game offers an extensive crafting system for engineers to supply weapons, "
+                   "armor, and ships to the galaxy. Develop new, cutting-edge armaments to give your "
+                   "side an edge, or open a shop in a bustling commercial district and become wealthy "
+                   "as part of a powerful engineering conglomerate."
+                   "\n\nL"
+                   "OTJ offers full PVP in both ground and space combat, governed by a set of rules to "
+                   "minimize griefing and ensure that all kills have sufficient in-character cause."
+                   "\n\n"
+                   "What role will you play? The legend awaits!");
+    }
+    if (hostUrl == QLatin1String("mume.org")) {
+        return qsl("Multi-Users in Middle-earth (MUME) is a highly competitive world PvP DikuMUD, set "
+                   "in J. R. R. Tolkien’s fictional world of Middle-earth, as described in The Hobbit "
+                   "and The Lord of the Rings, where players may choose to join the epic war between "
+                   "the forces of Sauron and the armies of the Free peoples. In MUME players can "
+                   "explore, role-play, acquire achievements, and complete quests across many "
+                   "challenging locations across Middle-earth such as Lothlórien, the Shire, Bree, "
+                   "Rivendell, Goblin-town, Mirkwood, Dol Guldur, and the Mines of Moria. The game is "
+                   "completely at no cost to play and has been continually enhanced since its "
+                   "inception in the fall of 1991.");
+    }
+    if (hostUrl == QLatin1String("mud.ren")) {
         return qsl("天下风云出我辈，一入江湖岁月催。\n"
-                "皇图霸业谈笑中，不胜人生一场醉。\n"
-                "提剑跨骑挥鬼雨，白骨如山鸟惊飞。\n"
-                "尘事如潮人如水，只叹江湖几人回。\n\n"
-                "中文开源武侠MUD炎黄群侠传，游戏包括25大门派和5大世家，正邪只在一念间；近千门武学等你学习，上百种任务随你体验；让自己成为一代宗师，江湖笑，恩怨了。");
-    } else {
-        return readProfileData(profile_name, qsl("description"));
-    }
+                   "皇图霸业谈笑中，不胜人生一场醉。\n"
+                   "提剑跨骑挥鬼雨，白骨如山鸟惊飞。\n"
+                   "尘事如潮人如水，只叹江湖几人回。"
+                   "\n\n"
+                   "中文开源武侠MUD炎黄群侠传，游戏包括25大门派和5大世家，正邪只在一念间；近千门武学等你学习，上百种任务随你体验；让自己成为一代宗师，江湖笑，恩怨了。");
```

Comment:
Please ask for English translation to deposit here as well.

Reply:
I had no idea what the translation was, I mentioned what I got from Google's translation when @vadi2 put that text in - as you picked up in https://github.com/Mudlet/Mudlet/pull/6140#issuecomment-1179811763 - OTOH there is no absolute reason why there will be a precisely equivalent English text - it isn't an English MUD. 🤷‍♂️

Reply:
They added a brief English description like Kung fu game, would you include it here as first paragraph?

Reply:
The game author is really uncomfortable with an English description - what would it add?

Reply:
Here is a brief one-line description they gave: https://github.com/Mudlet/Mudlet/pull/6140#issuecomment-1184121811

I think it helps English players gather the first of what happens in Mudlet still, not be dazzled by lots foreign language texts alone.

Reply:
TBH By definition a foreign language text will put off English players - which is actually appropriate IMVHO - and why should we be treating **English** players differently than say German or Russian ones? All in all I now think it would be reasonable to merge this PR in as it is.

\<aside>Perhaps we should be seeking out suitable description texts for all the predefined MUDs that do not currently have anything - for instance this would help to avoid *English* only speakers from clicking on Avalon without spotting that it is a German MUD!\</aside>